ui/package.json

```diff
@@ -30,6 +30,7 @@
     "i18next-browser-languagedetector": "^5.0.0",
     "lodash": "^4.17.21",
     "moize": "^5.4.7",
+    "mysql": "mysqljs/mysql",
```

Comment:
I believe you should use https://github.com/sidorares/node-mysql2 which is more up-to-date.

Reply:
Also this should be a dev dependency I guess, as it is test related.